ambari-server/src/main/python/ambari_server/dbConfiguration_linux.py

```diff
@@ -385,12 +385,24 @@ class PGConfig(LinuxDBMSConfig):
     PG_HBA_RELOAD_CMD = AMBARI_SUDO_BINARY + " %s reload %s" % (SERVICE_CMD, PG_SERVICE_NAME)
   else:
     SERVICE_CMD = "/usr/bin/env service"
-    PG_ST_CMD = "%s %s status" % (SERVICE_CMD, PG_SERVICE_NAME)
     if os.path.isfile("/usr/bin/postgresql-setup"):
         PG_INITDB_CMD = "/usr/bin/postgresql-setup initdb"
     else:
+      if OSCheck.is_suse_family() and not is_service_exist(PG_SERVICE_NAME):
+        versioned_script_paths = glob.glob("/usr/pgsql-*/bin/postgresql*-setup")
+        if versioned_script_paths:
+          for versioned_script_path in versioned_script_paths:
+            pgsql_version = re.search(r'pgsql-([0-9]+\.?[0-9]+)', versioned_script_path).group(1)
+            pgsql_service_file_name = "postgresql-%s" % pgsql_version
+            if is_service_exist(pgsql_service_file_name):
```

Comment:
i wonder what happen if there would be several services like postgres-9.5, postgres-9.6 and so on.   The glob may return result in random order and we will basically pick a random service instance. Should we decide, that we will pick latest version or first version and verify which of the services r in the enabled state?

Reply:
Good question, I've been thinking about it. I think we should let the users to decide that. If there are multiple service enabled then the user must specify which one to use, otherwise one of them will be chosen arbitrarily. 
So I just tried to follow the way we use at Redhat.

Reply:
well, for the usual installation we may ask user .. but for the silent, where no user interaction allowed - let's pick the latest enabled version. It should be reasonable choice

Reply:
ok then I'll change it